### ISSUE TITLE
Workshop

### ISSUE THREAD

**Issue by Jolsh2** [mdotstrange/NightmarePuppeteerPublic#299]
Tried Downloading jolshrio from workshop. previously i removed the mario body because of that body glitch with the level editor not loading properly. anyway it would not download the body for me. i had to manually download it from github and place it in the local drive. not sure if this is a steam issue or NP issue.

**Comment by mdotstrange** [comment]
Will check this out this weekend- thanks!

**Comment by SameSkeletonDifferentCloset** [comment]
> Tried Downloading jolshrio from workshop. previously i removed the mario body because of that body glitch with the level editor not loading properly. anyway it would not download the body for me. i had to manually download it from github and place it in the local drive. not sure if this is a steam issue or NP issue.

It still happens to me from time to time on the super edgy beta build.

**Comment by mdotstrange** [comment]
I will check out the workshop code for this and see what is going on- thanks.

**Comment by mdotstrange** [comment]
> Tried Downloading jolshrio from workshop. previously i removed the mario body because of that body glitch with the level editor not loading properly. anyway it would not download the body for me. i had to manually download it from github and place it in the local drive. not sure if this is a steam issue or NP issue.

When you say "tried downloading.. from workshop" What menu is that from? Did you upload a custom body to the "Other settings" menu and try to download there? Or was that body saved with a custom character- if you give me some more specifics I can check this out- thanks

**Comment by SameSkeletonDifferentCloset** [comment]
If I can chime in. It's the same thing I've encountered here and there. Whether downloading your uploaded characters from the workshop or moving them over locally to the NP folder, sometimes a body different than the Mario body will show. So you have to upload the body again. Downloading the saved workshop character or moving the backup over again will now show the Mario body. This can happen here and there. When I reformat my laptop I have to upload the Mario body again. So it's as if the body doesn't want to stay in NP for good.

**Comment by mdotstrange** [comment]
Ok so - like you manually added the mario body then saved a character with that body- then on reloading etc the mario body will be replaced by another?

**Comment by SameSkeletonDifferentCloset** [comment]
Yes. Exactly.

**Comment by mdotstrange** [comment]
Ok checking this out now- thanks

**Comment by mdotstrange** [comment]
I just tried adding the mario body- saving a custom character using the body- quit NP- relaunched and it did load my custom character with the mario body. I haven't tested a Workshop download with a custom body though- will have to test that next weekend-

**Comment by SameSkeletonDifferentCloset** [comment]
Yeah. It's something that only happens sometimes. Probably easier for users to just upload the body again through settings if it happens. 👍
